Mage/src/main/java/mage/choices/TwoChoiceVote.java

```diff
@@ -0,0 +1,64 @@
+package mage.choices;
+
+import mage.abilities.Ability;
+import mage.constants.Outcome;
+import mage.game.Game;
+import mage.players.Player;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * @author TheElk801
+ */
+public class TwoChoiceVote implements VoteHandler<Boolean> {
+
+    private final String choice1;
+    private final String choice2;
+    private final Map<UUID, List<Boolean>> playerMap = new HashMap<>();
+
+    public TwoChoiceVote(String choice1, String choice2) {
+        this.choice1 = choice1;
+        this.choice2 = choice2;
+    }
+
+    @Override
+    public void doVotes(Ability source, Game game) {
+        playerMap.clear();
+        for (UUID playerId : game.getState().getPlayersInRange(source.getControllerId(), game)) {
+            Player player = game.getPlayer(playerId);
+            if (player == null) {
+                continue;
+            }
+            boolean vote = player.chooseUse(Outcome.Neutral, "Vote", null, choice1, choice2, source, game);
```

Comment:
There is must be Outcome param, so it can be used by AI or AI hint code (see AI hint code example in that commit f692a1f09733b43ab9481f1e4727b02eae435d31).

Reply:
yeah, I'm already planning to change that code anyway